src/main/java/lottogame/ui/LottoOutputView.java

```diff
@@ -34,14 +36,28 @@ private static String formatLottoNumbers(List<LottoNumber> numbers) {
     public static void printWinningStatistics(Map<Rank, Integer> winningResults) {
         System.out.println("당첨 통계");
         System.out.println("---------");
-        for (Rank rank : Rank.values()) {
-            if (rank != Rank.NONE) {
-                System.out.printf("%d개 일치 (%.0f원) - %d개%n",
-                        rank.getRank(),
-                        rank.getAmount(),
-                        winningResults.getOrDefault(rank, 0));
-            }
+        getWinningRanks().forEach(rank -> printRankStatistics(rank, winningResults));
+    }
+
+    private static List<Rank> getWinningRanks() {
+        return Arrays.stream(Rank.values())
+                .filter(rank -> rank != Rank.NONE)
```

Comment:
객체에게 메시지를 보내는 방식으로 변경해보면 어떨까요?
```suggestion
                .filter(rank -> !rank.isNone())
```

Reply:
꼼꼼히 봐주셔서 감사합니다. 다음에는 놓치지 않겠습니다 :)